docs/getting-started/03-deploy-microservice.md

```diff
@@ -0,0 +1,83 @@
+---
+title: Deploy a microservice
+type: Getting Started
+---
+
+You will now start by deploying a standalone [`orders-service`](https://github.com/kyma-project/examples/blob/master/http-db-service/README.md) microservice in the `orders-service` Namespace. This microservice will act as a link between the external application and the Redis service and we will build the whole end-to-end flow around it.
+
+In this guide you will create:
+
+- Deployment in which you specify the configuration of your microservice.
+- Kubernetes Service through which your microservice will communicate with other resources on the cluster.
```

Comment:
```suggestion
- Kubernetes service through which your microservice will communicate with other resources on the cluster.
```
I believe `service` is not capitalized in our documentation. See the documentation [here](https://kyma-project.io/docs/components/application-connector/#architecture-application-connector-components-application-registry) and [here](https://kyma-project.io/docs/components/service-mesh/#overview-overview).
If it's our mistake, we should log an issue for that.

Reply:
Sure, in that case we should log a follow-up issue for updating our docs as k8s Service should be capitalized as all k8s resources.

Reply:
Done :) 
[Added](https://github.com/kyma-project/community/pull/504) it to the [guidelines](https://github.com/kyma-project/community/blob/master/guidelines/content-guidelines/08-style-and-terminology.md#capitalized-terms) and logged an [issue](https://github.com/kyma-project/kyma/issues/9368) for that.